src/AutoCrudAdmin/Helpers/Implementations/FormControlsHelper.cs

```diff
@@ -9,6 +9,7 @@ namespace AutoCrudAdmin.Helpers.Implementations
     using AutoCrudAdmin.Extensions;
     using AutoCrudAdmin.ViewModels;
     using Microsoft.EntityFrameworkCore;
+    using OJS.Data.Models.Users;
```

Comment:
AutoCrudAdmin should not reference OJS. The project must be buildable on its own.

Reply:
Reference removed.